include/evmc/evmc.hpp

```diff
@@ -27,7 +27,17 @@ class result : private evmc_result
     using evmc_result::status_code;
 
     /// Converting constructor from raw evmc_result.
-    explicit result(evmc_result const& res) noexcept : evmc_result{res} {}
```

Comment:
Do we consider this as a breaking change?

Reply:
It is. Can be moved to 6.3 but as we are fixing C++ API in 6.2 anyway I think we can also include this change.

Reply:
I don't think this is fixing a bug in 6.2.x, but improving the API. Following that logic it should be added to 6.3.x, if I'm completely pragmatic.